src/providers/bsky/SocialMedia.ts

```diff
@@ -84,13 +88,17 @@ export class BskySocialMedia implements Provider {
     }
 
     async deletePost(postId: string): Promise<boolean> {
-        throw new NotImplementedError();
+        postId = decodeURIComponent(postId);
```

Comment:
```suggestion
        const atUri = resolvePostAtUri(postId);
```

For `postId`, let it be `[userid]-[recordid]`.

AT URI: `at://did:plc:3t3i7gsjdvirweekn5ygx4z5/app.bsky.feed.post/3lgztlibevs2y`
PostId: `3t3i7gsjdvirweekn5ygx4z5-3lgztlibevs2y`